ProjectLighthouse.Servers.GameServer/Controllers/Slots/ScoreController.cs

```diff
@@ -78,6 +79,7 @@ public async Task<IActionResult> SubmitScore(string slotType, int id, [FromQuery
         if (slotType == "developer") id = await SlotHelper.GetPlaceholderSlotId(this.database, id, SlotType.Developer);
 
         score.SlotId = id;
+        if (childId != 0) score.ChildSlotId = childId;
```

Comment:
```suggestion
        score.ChildSlotId = childId;
```
Since we made the ChildSlotId column not nullable this shouldn't be necessary anymore, if childId is null it will just set it to 0 and it makes the code look cleaner